src/gui/widgets/FloatModelEditorBase.cpp

```diff
@@ -439,9 +442,16 @@ QString FloatModelEditorBase::displayValue() const
 	if (isVolumeKnob() &&
 		ConfigManager::inst()->value("app", "displaydbfs").toInt())
 	{
-		return m_description.trimmed() + QString(" %1 dBFS").
-				arg(ampToDbfs(model()->getRoundedValue() / volumeRatio()),
-								3, 'f', 2);
+		auto const valueToVolumeRatio = model()->getRoundedValue() / volumeRatio();
+		if (valueToVolumeRatio == 0.)
+		{
+			return m_description.trimmed() + QString(" -∞ dBFS");
+		}
+		else
+		{
+			return m_description.trimmed() + QString(" %1 dBFS").
+				arg(ampToDbfs(valueToVolumeRatio), 3, 'f', 2);
+		}
```

Comment:
What do we think of this instead?
```suggestion
		return m_description.trimmed() + (
			valueToVolumeRatio == 0.
			? QString(" -∞ dBFS")
			: QString(" %1 dBFS").arg(ampToDbfs(valueToVolumeRatio), 3, 'f', 2)
		);
```

Reply:
Looks better! Suggestion applied with commit bb14d25c853.